### PR TITLE
[#159067640] Track Redis communication errors using logger

### PR DESCRIPTION
**Pull request by LsKoder**
Track Redis communication errors using logger

### CODE REVIEW

File: src/App.ts [hunk 1]
```diff
@@ -131,25 +131,39 @@ function setRestfulRoutes(
...
+  })();
+
+  redisClient.on("error", err => {
+    logger.error(`REDIS Connection error: ${err}`);
+  });
+  redisClient.on("warning", err => {
+    logger.warn(`REDIS Connection warning: ${err}`);
+  });
+  redisClient.on("end", () => {
+    logger.error(`REDIS Connection lost`);
```
Comment (cloudify) [inline src/App.ts]: I'm not sure this is an error, you probably get this event even when the client closes the connection... @gunzip ?

Comment (gunzip) [reply src/App.ts]: I've no clue but I guess so

Comment (LsKoder) [reply src/App.ts]: If a connection cannot be enstablished but server is available or if connection is lost, it just trigger an "end" event. I tested both cases during my tests.
Connection should never be closed when server is running, so it should be logged as error in both cases.

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#159067640](https://www.pivotaltracker.com/story/show/159067640): Tracciare errori di comunicazione con Redis su PagoPaProxy

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**